source/uwp/AdaptiveCardsObjectModel/dll/dll.cpp

```diff
@@ -0,0 +1,57 @@
+//
+//    Copyright (C) Microsoft.  All rights reserved.
+//
+#include "pch.h"
+#include <windows.h>
+#include <wrl.h>
+#include <wrl\wrappers\corewrappers.h>
+
+using namespace Microsoft::WRL;
+
+//WrlCreatorMapIncludePragma(AdaptiveCardsObjectModel);
```

Comment:
I.... don't know. This project is a copy of the renderer project and this exists there (uncommented). I uncommented it here as well in case it's doing something important for some projection or scenario that our tests don't cover.

Reply:
Removed